src/main/java/htsjdk/samtools/SBIIndex.java

```diff
@@ -0,0 +1,300 @@
+package htsjdk.samtools;
+
+import htsjdk.samtools.util.BinaryCodec;
+import htsjdk.samtools.util.BlockCompressedFilePointerUtil;
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * SBI is an index into BGZF-compressed data files, which has an entry for the file position of the
+ * start of every <i>n</i>th record. Reads files that were created by {@link BAMSBIIndexer}.
+ */
+public final class SBIIndex implements Serializable {
```

Comment:
If this will be in htsjdk, add a TODO pointing to the PR to be removed once htsjdk is upgraded with that change.

Reply:
Good idea - I'll do that.

Reply:
Done